README.md

```diff
@@ -36,15 +38,33 @@ docker run -p 80:8080 \
 cnwan_adaptor
 ```
 
+## Quickstart
+If you want a minimal working setup, the script [setup_kubecon_demo.sh](examples/setup_kubecon_demo.sh) sets everything up a for you in the adaptor and your SD-WAN controller. Before running it, please:
 
+* Install the bash utility `jq`
+* Edit the file [policies_definition.json](examples/policies_definition.json) with the tunnels, vpns and sites of your deployment. You can add as many polcies and SLAs as you need. In addition, specify the metadata keys and values used by the CNWAN reader.
```

Comment:
To me saying "edit the file" sounds like there are things that need to be changed. We could say "Take a look a the policies_definition.json file and adapt it to your environment (tunnels, VPNs and deployment sites). The default values will re-create the [CN-WAN demo presented at KubeCon EU 2020](https://www.cisco.com/c/en/us/training-events/events/kubecon-europe.html#~demos-and-presentations)."